workflows/differentialabundance.nf

```diff
@@ -74,12 +74,36 @@ if (run_gene_set_analysis) {
     } else if (params.gprofiler2_run) {
         if (!params.gprofiler2_token && !params.gprofiler2_organism) {
             error("To run gprofiler2, please provide a run token, GMT file or organism!")
-        } else {
-            ch_gene_sets = [[]]     // For gprofiler2 which calls ch_gene_sets.first()
         }
     }
 }
 
+// Define tool channel
```

Comment:
Comment should be for understanding the code- people don't know what you mean by multitool logic